spec/unit/spark_api/models/saved_search_spec.rb

```diff
@@ -139,18 +160,14 @@
   describe "can_have_newsfeed?" do
```

Comment:
The changes below are unrelated, but I had the file open so I figured I might as well fix these things.